backend/substrapp/serializers/ledger/computeplan/serializer.py

```diff
@@ -75,6 +81,8 @@ def get_args(self, data):
             }
             if 'in_models_ids' in data_traintuple:
                 traintuple['inModelsIDs'] = data_traintuple['in_models_ids']
+            if 'metadata' in data_traintuple:
+                traintuple['metadata'] = data_traintuple['metadata']
```

Comment:
Is there a reason to only include metadata if present vs always including it like we do for the other assets (datasets, objectives etc.) ?

Reply:
Well, it doesn't work when I change this, so I suppose it's the right way to do it.